docker/deployed/staging/api/config.json

```diff
@@ -39,7 +39,8 @@
     "Pruning": {
       "Enabled": true,
       "KeepFiles": 5
-    }
+    },
+    "URL" : "https://storage.googleapis.com/tableland-staging/backups/tbl_backup_latest.db.zst"
```

Comment:
Points to the latest staging backup file. I uploaded this manually. Later will have an automatic process.